modules/phase_field/src/base/PhaseFieldApp.C

```diff
@@ -218,9 +219,11 @@ PhaseFieldApp::PhaseFieldApp(const InputParameters & parameters) :
     MooseApp(parameters)
 {
   Moose::registerObjects(_factory);
+  TensorMechanicsApp::registerObjects(_factory);
   PhaseFieldApp::registerObjects(_factory);
 
   Moose::associateSyntax(_syntax, _action_factory);
+  TensorMechanicsApp::associateSyntax(_syntax, _action_factory);
```

Comment:
Uhm, I has a PR a week ago that removed that (where I moved the tensors into framework). Let me check.